packages/codemods/vite.config.mts

```diff
@@ -0,0 +1,17 @@
+import { defineConfig, configDefaults } from 'vitest/config'
+
+export default defineConfig({
+  test: {
+    testTimeout: 20_000,
+    exclude: [
+      ...configDefaults.exclude,
+    '**\/__fixtures__/**',
+    '**\/__testfixtures__/**',
+    '**\/__tests__/utils/**',
+    '.d.ts',
+    'dist',
```

Comment:
Tiniest of things but from what I have seen [here](https://vitest.dev/config/#exclude) the vitest defaults already exclude dist.